### PR TITLE
{vis}[GCC/13.2.0] GST-plugins-base v1.24.8, GStreamer v1.24.8, Paraver v4.11.4, wxWidets v3.2.6

### PR DESCRIPTION
**Pull request by julianmorillo**

### CODE REVIEW

File: easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -26,7 +26,7 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('GLib', '2.78.1'),
+    ('GLib', '2.80.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-13.2.0.eb:29]: There should be no need for this (and it'll cause fallout for sure)...

**Review by boegel — DISMISSED** [review]

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 24.04.1 LTS (Noble Numbat), x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.12.3
See https://gist.github.com/bedroge/baff3028ac2dc8549728adcfb611c24c for a full test report.

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.10, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/8a5bf72e138628d82d70c5d48594e41b for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @julianmorillo!

**Pull request merged**